Tests/TuistGeneratorTests/Generator/Mocks/MockSchemesGenerator.swift

```diff
@@ -5,9 +5,23 @@ import TuistCoreTesting
 @testable import TuistGenerator
 
 final class MockSchemesGenerator: SchemesGenerating {
+    
+    var generateWorkspaceSchemeArgs: [(workspace: Workspace, xcworkspacePath: AbsolutePath, generatedProjects: [AbsolutePath: GeneratedProject], graph: Graphing)] = []
+    
     var generateProjectSchemeArgs: [(project: Project, xcprojectPath: AbsolutePath, generatedProject: GeneratedProject, graph: Graphing)] = []
     
+    
+    
+    func generateWorkspaceSchemes(workspace: Workspace,
+                                  xcworkspacePath: AbsolutePath,
+                                  generatedProjects: [AbsolutePath: GeneratedProject],
+                                  graph: Graphing) throws {
+        generateWorkspaceSchemeArgs.append((workspace: workspace, xcworkspacePath: xcworkspacePath, generatedProjects: generatedProjects, graph: graph))
+    }
+    
     func generateProjectSchemes(project: Project, xcprojectPath: AbsolutePath, generatedProject: GeneratedProject, graph: Graphing) throws {
         generateProjectSchemeArgs.append((project: project, xcprojectPath: xcprojectPath, generatedProject: generatedProject, graph: graph))
     }
+    
+    func wipeSchemes(at: AbsolutePath) throws { }
```

Comment:
Although we don't have tests that check how many times this method is called and with which arguments, I'd add a `wipeSchemesArgs` variable to the mock where we store that information. It might be useful in the future.